site/docs/guidelines/tables-lists.mdx

```diff
@@ -0,0 +1,96 @@
+---
+title: "Tables vs lists"
+navTitle: "Tables vs lists"
+summaryParagraph:
```

Comment:
Just checking that you've deliberately left the summary paragraph blank? I wonder if you could pull the 'General' stuff up here. Maybe something like "Lists and tables can be quite visually similar and can sometimes represent the same data, but they have distinct patterns which shouldn't be mixed."

Reply:
Love it 👍